test/smoke/harness/graphql.go

```diff
@@ -24,18 +24,23 @@ const DefaultGraphQLAdminUserID = "00000000-0000-0000-0000-000000000002"
 
 func (h *Harness) insertGraphQLUser(userID string) string {
 	h.t.Helper()
-	var err error
 	if userID == DefaultGraphQLAdminUserID {
 		permission.SudoContext(context.Background(), func(ctx context.Context) {
-			_, err = h.backend.UserStore.Insert(ctx, &user.User{
-				Name: "GraphQL User",
-				ID:   userID,
-				Role: permission.RoleAdmin,
-			})
+			foundUser, err := h.backend.UserStore.FindOne(ctx, userID)
+			if foundUser == nil {
```

Comment:
This creates a bit of a race condition -- maybe separate the insert from the session creation